### PR TITLE
Adding multinest

### PR DESCRIPTION
**Pull request by giacomov**
Adding the Multinest package, a Bayesian sampler to compute evidence: https://github.com/JohannesBuchner/MultiNest

### CODE REVIEW

**Comment by mwcraig** [comment]
@giacomov -- thanks for opening the PR; I have some concerns related to the distribution of this package based on the github repos of it, which do not appear to be the "official" source (https://ccpforge.cse.rl.ac.uk/gf/project/multinest/)

@conda-forge/core -- this is interesting...the github repo appears to be a clone of the source derived from ccpforge. The license is a custom one (https://github.com/JohannesBuchner/MultiNest/blob/master/LICENCE) and it is not clear to me whether that distribution is persmissible under 4c and 4d.

@giacomov -- can you email the authors of multinest to ask them for permission to redistribute it via the github repo and via conda-forge?

**Comment by giacomov** [comment]
Regarding the repository on github, it is from the same author as the sourceforge repository and it is identical. The problem with the sourceforge one is that it cannot be downloaded on the fly, because it is password protected.

Regarding the license, I will write them. I thought including the license as they describe it was enough, but I can double check with the author, which I meant to do anyway.

I'll keep you posted.

**Comment by giacomov** [comment]
I just heard back from the owner of Multinest, he is fine with us releasing it on Conda, I'll add him as maintainer. This is his answer:

```Hi Giacomo,

Thanks for your work on preparing the recipe for MultiNest to be included in conda-forge. I would be happy for you to include MultiNest in conda-forge. I think including the LICENCE file should be fine. Please do add me as an admin.

Thanks,
Farhan```

You can go ahead and merge this pull request at your earliest convenience.

Thanks!

File: recipes/multinest/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+package:
+  name: multinest
+  version: "3.10"
+
+source:
+  git_url: https://github.com/JohannesBuchner/MultiNest
```
Comment (isuruf) [inline recipes/multinest/meta.yaml]: https://conda-forge.org/docs/meta.html#build-from-tarballs-not-repos

**Pull request merged**